info.go

```diff
@@ -540,28 +640,25 @@ func (pi *ProgramInfo) KsymAddrs() ([]uintptr, bool) {
 // ErrNotSupported if the program was created without BTF or if the kernel
 // doesn't support the field.
 func (pi *ProgramInfo) FuncInfos() (btf.FuncOffsets, error) {
-	id, ok := pi.BTFID()
-	if pi.numFuncInfos == 0 || !ok {
-		return nil, fmt.Errorf("program created without BTF or unsupported kernel: %w", ErrNotSupported)
-	}
-
-	h, err := btf.NewHandleFromID(id)
-	if err != nil {
-		return nil, fmt.Errorf("get BTF handle: %w", err)
-	}
-	defer h.Close()
+	var funcs btf.FuncOffsets
+
+	err := pi.withBTFSpec(nil, func(spec *btf.Spec) error {
+		var err error
+		funcs, err = btf.LoadFuncInfos(
+			bytes.NewReader(pi.funcInfos),
+			internal.NativeEndian,
+			pi.numFuncInfos,
+			spec,
+		)
+		return err
+	})
 
-	spec, err := h.Spec(nil)
-	if err != nil {
-		return nil, fmt.Errorf("get BTF spec: %w", err)
-	}
+	return funcs, err
+}
 
-	return btf.LoadFuncInfos(
-		bytes.NewReader(pi.funcInfos),
-		internal.NativeEndian,
-		pi.numFuncInfos,
-		spec,
-	)
+// JitedInfo returns the JITed information of the program.
+func (pi *ProgramInfo) JitedInfo() *ProgramJitedInfo {
+	return pi.jitedInfo.clone()
 }
```

Comment:
This isn't inline with the rest of the API. Please take a look at the comment at the start of the file. Lets add these new fields with getters as well.

Reply:
> Please take a look at the comment at the start of the file. Lets add these new fields with getters as well.

Thank you for your reminder. I'll do it.